lib/src/working_copy.rs

```diff
@@ -795,7 +795,7 @@ impl TreeState {
                             }
                         }
                     } else {
-                        let work_item = WorkItem {
+                        let file_to_visit = FileToVisit {
```

Comment:
It looks like it's still always a directory, so renaming it doesn't make sense to me at this point